core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -350,6 +350,18 @@
   MASTER_METADATA_SUSPENDABLE("master.metadata.suspendable", "false", PropertyType.BOOLEAN,
       "Allow tablets for the " + MetadataTable.NAME
           + " table to be suspended via table.suspend.duration."),
+  MASTER_STARTUP_TSERVER_AVAIL_MIN_COUNT("master.startup.tserver.avail.min.count", "0",
+      PropertyType.COUNT,
+      "Minimum number of tservers that need to be registered before master will "
+          + "start tablet assignment - checked when master gets lock"),
+  MASTER_STARTUP_TSERVER_AVAIL_ABORT_ENABLED("master.startup.tserver.avail.abort.enabled", "false",
```

Comment:
What use case does this option satisfy?  I am trying to think through how a user would use this option.